src/wp-admin/includes/media.php

```diff
@@ -3843,3 +3843,12 @@ function wp_media_attach_action( $parent_id, $action = 'attach' ) {
 		exit;
 	}
 }
+
+/**
+ * Returns the default image output mapping.
```

Comment:
Suggesting a rewording here. Also, while I agree the function name should be decoupled from the specifics of JPEG and WebP, I think it would still be good to explain that here. Maybe something like this?
```suggestion
 * Filters the default image output mapping.
 *
 * With this filter callback, WebP image files will be generated for certain JPEG source files.
```

Feel free to reword further, but IMO worth highlighting this somehow.